server/xpub/entities/file/data-access.test.js

```diff
@@ -0,0 +1,76 @@
+const dataAccess = require('./data-access')
+const manuscriptTestData = require('../manuscript/data-access.test.data')
+const { createTables } = require('@pubsweet/db-manager')
+const FileManager = require('./index')
+
+let testId = 0
+let manuscriptId = 0
+
+async function createAFile(mid, fileName) {
+  const f = FileManager.new()
+  f.manuscriptId = mid
+  f.filename = fileName
+  return dataAccess.insert(f)
+}
+
+const initializeDatabase = async () => {
+  await createTables(true)
+  manuscriptId = await manuscriptTestData.createSingleManuscript()
+  testId = await createAFile(manuscriptId, 'testFile')
+  console.log('Created testId', testId)
+
+  return testId
+}
+
+describe('FileAccessLayer', () => {
+  beforeEach(async () => {
+    testId = await initializeDatabase()
```

Comment:
You're already assigning `testId` to the outer variable in `initializeDatabase`.